test/Mechanical/rotational.jl

```diff
@@ -98,4 +98,77 @@ end
     # sol = solve(prob, Rodas4())
 
     # Plots.plot(sol; vars=[inertia2.w, inertia3.w])
+end
+
+@testset "Stick-Slip" begin
```

Comment:
I also added a rotational friction component and an interesting test for it:
![stick_slip](https://user-images.githubusercontent.com/50108075/167302268-c93113ed-f0f7-49bc-999a-d837bef01c25.png)

Since there is no event system working reliably, I used smooth approximations for the Stribeck and Coulomb friction. The test case is from [Matlab](https://www.mathworks.com/help/physmod/simscape/ug/mechanical-rotational-system-with-stick-slip-motion.html).

Reply:
Is the jagged "inertia" profile a sign of stick slip? If so, isn't it strange that it appears without the velocity being close to zero?

Reply:
Yes, relative velocity. The plot is a bit misleading.

Reply:
![stick_slip](https://user-images.githubusercontent.com/50108075/169659090-1427d581-2c1b-4b3f-8f1f-17b602463d7d.png)

Reply:
Could you try `plot(..., denseplot=false)`? https://diffeq.sciml.ai/stable/basics/plot/#Density

Reply:
I can't see a difference:

![stick_slip](https://user-images.githubusercontent.com/50108075/170975865-035d499d-e8f7-47e0-ab0a-d730882acf47.png)

May I ask what you are after?

Reply:
Sometimes the interpolation misbehaves, with `denseplot=false` it will only plot the points that solvers step on.